gcc/rust/typecheck/rust-hir-type-check-expr.cc

```diff
@@ -1277,7 +1277,8 @@ TypeCheckExpr::visit (HIR::BreakExpr &expr)
 {
   if (!context->have_loop_context ())
     {
-      rust_error_at (expr.get_locus (), "cannot %<break%> outside of a loop");
+      rust_error_at (expr.get_locus (), ErrorCode ("E0268"),
+		     "%<break%> outside of a loop or labeled block");
```

Comment:
Changed error message, similiar to rustc- [`E0268`](https://doc.rust-lang.org/error_codes/E0268.html).
- You can see it on [`godbolt`](https://godbolt.org/#z:OYLghAFBqd5QCxAYwPYBMCmBRdBLAF1QCcAaPECAMzwBtMA7AQwFtMQByARg9KtQYEAysib0QXACx8BBAKoBnTAAUAHpwAMvAFYTStJg1DEArgoKkl9ZATwDKjdAGFUtEywZ7HAGTwNMAHLuAEaYxHoADqgKhHYMLm4ekdGxAr7%2BQSyh4VyWmNa2AkIETMQECe6euVaYNnHFpQTpgSFheuaNFUnVJWXNmdkSAJSWqCbEyOwcAPTTANQIBAQRCiCz6KjIAHSm5gC0BkZbJMDTYcQkAPpoWArT2BoATABsABxbiyy0AKQaAILfR4AZjAYG%2BAFYAEJiWioADuEBMDDMmHQQwhABFfn8qAw5sFUKgIEM5t8AOyQ7FzalzNCCPwmTDfIGQuazUkUuboYB7c4kUmPR5bOm2ZGYLZzMYEGJYSVUOZMOaw1ARQGPDlY/40/HETBMADWzNZ7PJrO5vOIF2IAqFwV1BolUplmDlCqVhIikutBlC9HQ%2BNhyENgo12PJmoB/1xcxYTD8xI5lK1NIJRPR/3DHBGtE44N4ng4WlIqE4ACUzAQ5goxhMXYCgTxSARNFmRvqQMCtpJwUDJGSG0DgWTHlxHvpOJJ8y3i5xeKsNE2WyM4LAYIgUKgWBE6GFyJQ0Fud%2BFgMhkMQFHsFMwVghUBYsAA3PCTABqeEwcIA8hFGJxGzRaAIMJVggYJp2CPxSgATz/XgIOYYgoM/YJtFqZtuF4A82EET8GFoGDC14LBghMYAnBhVYMNILBYyMcRCOovBdTqB9MEootMFUWoTCA2DyEEfJp1oPA7WglwsGnAhiDwFhYJGKgDGABQ3w/b9fyo/hBBEMR2CkGRBEUFR1AY3RckOYwK30ETVkgEYVUKBhKN4VBWMtPBbngCBmDYEBUVSBhSAfMRGU4R4nh4IYRhqOp7AgRwuiqUgfD8FosjaXIohiByEuSLK4n6VocjyAp6l6cpXEqdp8jQ0rGgKtKio6Mocp6OqUoGdKoprSZhnHDg81IAsixLDg5lUV5nj2Z5JDmABxJwnFLIQ5ggebFuW2NzDCEkIFwQh%2BXrLghl4dCtEi0h23BBccw4SdSFkyQAE4tlecFnhHLhniBZ4NFex7nkG6cRrnEAF1OrNSBXdcpQiHi9wgA9t3oYgAlYKZdgIZA5i4LY3i2Dj8CIaT0D0TThFEcQ9LJwy1GnUzSDhYgmAiOS%2BoGobnM4T8eNhytUHlcbJumuaFqWla1rFzagOIHaXEPZGBQbY7F0I86bru2Srq2KQgTJV5Xm%2BoFHqkDRpA5mcOBBsGlzbDsyS7XWyW7J3Tce14NHevqgSnBjgZVs7IbXKGICQRGj3hsPkZAYgpEehdH2fTAVK/H8C3/OhpZAsCGPg6C%2BNzxDkNQmw%2BKwxgCFw/Dp2I0jyNoWhKMbGjDGAeii3wZjbFY9jeE47jeKovwgJuothNExDxKmIspJk1mFKYJTk7UtPeDJ7TKekamlFpkyO30Fvo8ssebOJYsIgcpzi1c4m2PgKLqpizw4oYZwKu6JKX/qwYMpSbK38SzK/kv7pWKjVIoZUWqgMfg0Po7VCrtAgf/BBbUMjwKOqMcYPV0E3XZkDTgcwMZYxxk9fGK09pE0VkdE6NtSAID1FgcIp92xcA0NdCcvBNYLnNn7ec/sIbByQDDOGFAEabiRmEVGPkOCCymjNCWy1Vqiw2kwLaMte6ExIO5Umsh166U3rIGmxkiy6DHIzZmrMcE%2B2GlzHmPFXSEOxl2Z6GgVpy3Edaesjxlbg3OnQpgDDKC2xYWw26HCQBXUBr7WclhQZ8POu2R4kgtjgnBJII2Gg3pAjepIV4bsvZWM5pbOJgdPJB1DmI8OIjI5tBjmSVh1FMBPlfO%2BFO6l06AWApQbORYC4EUbL0ouaFS6bmwhXPCBF26YBImRCifFm50SnkRJiNVu7Tj7sgHiUxGxD0EgxMeTMJ4YEWU2aSskMLyUUspFpy8%2BJrwpno/S8ht5GJ0CAIE%2B8jCH3MFZYIJ87LnziJfFy5x3K31slAhyDgX6QOSqghquV/KQMAQ5YBjUH4ORgeVRIiVooYrKqi5BzUkGtVgXC7%2BXVMG6WzLmApFsCEViIVsOppDdoaI8cCKhcSRh%2BICUwjsSSUlpLjpk7JuTXh9Q1uErheCim8J8SU2AgjbEWCqRU5Gkj0YMscSQgm%2B1ibaK0vciQ%2BiDLPLpm8hmTMWbnLZrSka3Nlh2P5vS8wjLmUuIgG4o8isgTeJoTytop91ZhIidw6JcqaEJIFak9JIrppivyZE6xRSfHUo4I8O10TU2BWAnEEAkggA%3D%3D)

Reply:
- rustc 1.49 has error message:
```rust
`break` outside of a loop
```
- But, I updated the message according to rustc nightly version

```rust
`break` outside of a loop or labeled block
```
- You can see difference on the `compiler-explorer` on the link above.